census/templates/homepage.html

```diff
@@ -98,12 +98,12 @@ <h2 class="column-full subhead">Exploring data</h2>
         <header class="column-full">
             <h1 class="article-header">About Wazimap</h1>
 
-            <p class="explainer">Wazimap, from the Xhosa word <i>ulwazi</i> for knowledge, is a joint project by <a href="http://www.mediamonitoringafrica.org/" target="_blank">Media Monitoring Africa</a> and <a href="http://www.code4sa.org/" target="_blank">Code for South Africa</a> that provides easy access to <a href="http://www.statssa.gov.za/" target="_blank">South African census</a> and <a href="http://www.elections.org.za/" target="_blank">elections</a> data.</p>
+            <p class="explainer">Wazimap, from the Xhosa word <i>ulwazi</i> for knowledge, is a joint project by <a href="http://www.mediamonitoringafrica.org/" target="_blank">Media Monitoring Africa</a> and <a href="https://www.openup.org.za/" target="_blank">OpenUp</a> that provides easy access to <a href="http://www.statssa.gov.za/" target="_blank">South African census</a> and <a href="http://www.elections.org.za/" target="_blank">elections</a> data.</p>
 
             <p>Read more about <a href="/about">who we are and why we built Wazimap</a>.
 
             <h2 class="column-full subhead">Shoulders of Giants</h2>
-            <p>Wazimap is based on the amazing <a href="http://censusreporter.org/" target="_blank">Census Reporter</a> and is completely open source. You can <a href="https://github.com/Code4SA/censusreporter/">fork Wazimap on Github</a> and build one for your country.</p>
+            <p>Wazimap is based on the amazing <a href="http://censusreporter.org/" target="_blank">Census Reporter</a> and is completely open source. You can <a href="https://github.com/OpenUpSA/censusreporter/">fork Wazimap on Github</a> and build one for your country.</p>
```

Comment:
Woah, this is old. Let's change ``You can fork ...``  to:

You can [build a Wazimap for your own country](http://wazimap.readthedocs.io/en/latest/).